kiwi/solverimpl.h

```diff
@@ -382,24 +383,22 @@ class SolverImpl
 	for tracking the movement of the constraint in the tableau.
 
 	*/
-	Row* createRow( const Constraint& constraint, Tag& tag )
+	std::unique_ptr<Row> createRow( const Constraint& constraint, Tag& tag )
 	{
-		typedef std::vector<Term>::const_iterator iter_t;
 		const Expression& expr( constraint.expression() );
-		Row* row = new Row( expr.constant() );
+		std::unique_ptr<Row> row( new Row( expr.constant() ) );
```

Comment:
Really wish we had `std::make_unique()` here, but it's since c++14 :|

Reply:
Agree on that, I also wanted to make `strength::create` constexpr but std::max and std::min are not constexpr in C++11 and I though making custom constexpr max/min functions was a bit overkill.